integration/docker/diff_test.go

```diff
@@ -28,7 +28,7 @@ var _ = Describe("diff", func() {
 
 	BeforeEach(func() {
 		id = randomDockerName()
-		_, _, exitCode := DockerRun("--name", id, "-d", Image, "sh")
+		_, _, exitCode := DockerRun("--name", id, "-di", Image, "sh")
```

Comment:
The fix looks like it should work - but, is it the right fix?
Given we don't actually use the `sh` afaict, I suspect we'd be more 'correct' launching a benign permanent process here - my normal choice is `tail -f /dev/null` for instance.
I don't know if there is a docker idiom for doing this - for having a pre-made container sat waiting to run `exec`s - anybody? This feels awfully like our 'pause' container :-)

Reply:
I agree that `tail -f /dev/null` would signal the intent of the test more clearly, so would indeed be better.

But this change is the minimal fix for the test. How about a compromise where @sboeuf adds a comment explaining why the `-i` is being used?

Reply:
I am okay modifying the command instead of the flag here. The goal is to maintain the container running, I don't care the way we do this.